style.css

```diff
@@ -0,0 +1,63 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: url("https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg");
+  padding: 50px 0;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+#title,
```

Comment:
- [ ] Maybe you could use opacity for the background to this title and description. Doing this, I think it will help to improve the readability of your website 😍